newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -226,7 +226,14 @@ export const HomePage = React.memo<Props>(
         displayTooltipDelayed,
         setDisplayTooltipDelayed,
       ] = React.useState<boolean>(false);
-      const { games, gamesFetchingError, fetchGames } = useGamesList();
+      const {
+        games,
+        gamesFetchingError,
+        fetchGames,
+        onGameUpdated,
+      } = useGamesList();
+      const openedGame =
+        (games && games.find(game => game.id === openedGameId)) || null;
```

Comment:
This is the fix

Reply:
This `openedGame` could probably be in a useMemo() to avoid making that find() on every render